src/main/java/com/minecolonies/apiimp/initializer/InteractionValidatorInitializer.java

```diff
@@ -129,7 +130,7 @@ public static void init()
         InteractionValidatorRegistry.registerStandardPredicate(new TranslationTextComponent(COM_MINECOLONIES_COREMOD_ENTITY_WORKER_INVENTORYFULLCHEST),
           citizen -> citizen.getWorkBuilding() != null && InventoryUtils.isProviderFull(citizen.getWorkBuilding()));
         InteractionValidatorRegistry.registerPosBasedPredicate(
-          new TranslationTextComponent(BUILDING_LEVEL_TOO_LOW), (citizen, pos) ->
+          new TranslationTextComponent(RequestSystemTranslationConstants.REQUEST_SYSTEM_BUILDING_LEVEL_TOO_LOW), (citizen, pos) ->
```

Comment:
the class has RS in name, might want to remove rs from constant name then (look for other constant also if so)